src/components/BuildArtifactsPage/BuildArtifactsPage.tsx

```diff
@@ -49,6 +49,10 @@ export const BuildArtifactsPage = ({ componentId = 'a1' }: IBuildArtifactsPagePr
             buildArtifactsCount={serviceContainerArtifacts.data?.content?.length}
           />
         </ToolbarItem>
+        <Text>
+          This list contains artifacts produced by this Build. Artifact is represented by PNC Identifier. You can change quality
```

Comment:
(optional) `... Eacg Artifact is represented by a PNC Identifier. You can also change the quality of all artifacts produced by the Build here.`